src/components/app.js

```diff
@@ -7,10 +7,20 @@ const App = () => {
   const [currentPlan, setPlan] = useState("");
   const [modal, setModal] = useState(false);
   const [jsonText, setJSONText] = useState("");
+  const [isCustomAvailable, setIsCustomAvailable] = useState(false);
   useEffect(() => {
     let attr = window.location.search;
     let year = "";
     let fastTrack = false;
+    // Get custom json plan from localstorage
+    const customJSONText = localStorage.getItem("customJSON");
```

Comment:
I think this part of code should be in the else part of `if(attr.length !=0)` to make the priority of query string more than saved value in local storage.